scripts/linters/general_purpose_linter.py

```diff
@@ -569,19 +583,26 @@ def check_file_type_specific_bad_pattern(filepath, content):
         content: str. Contents of the file.
 
      Returns:
-        failed: bool. True if there is bad pattern else false.
-        total_error_count: int. The number of errors.
+        tuple(failed, total_error_count, summary_messages). A 3-tuple whose
```

Comment:
Ditto as above add type not variable name. see other docstring for references.

Reply:
Done.